Mechanical/Differential Pivot/README.md

```diff
@@ -1,38 +1,147 @@
-| Group/Organization/School | JPL                |
-| Time to build             | 2 hrs              |
+## Machining/Fabrication
 
-## Skills
-  * Band Saw/Dremel
-  * Drill Press/Hand Drill
+1. Aluminum Rods: Cutting
 
-## Tools
+    Parts/Tools Necessary
 
-### Mandatory 
-  * Band saw or Dremel
-  * Drill press or Hand Drill
-  * Allen Key set
-  * Imperial Wrench Set
+    | Item | Ref | Qty | Image |
+    | :--- | :-- | :-- | :---: |
+    | 0.5" x 3' Aluminum Tube | S16 | 1 | ![S16](/images/components/Structural/S16.png) |
+    | 0.5" x 4" Aluminum Tube | S18 | 4 | ![S18](/images/components/Structural/S18.png) |
+    | Metal Hacksaw or Bandsaw | D4 | 1  | ![D4](/images/components/Tools/D4.png) |
+    | Vice clamp or C clamps | D5 | 1 | ![D5](/images/components/Tools/D5.png) |
 
-### Optional
+    1. Take the 3 foot piece of 0.5 inch aluminum rod S16 and cut it into one 15 inch piece and one 13.5 inch piece. These will now be referred to as the parts S16A and S16B respectively. In addition, take two of the 4 inch aluminum rods S18 and cut them down to 3 inches in length as well.
 
-## Dependencies on Other Modules:
-  * None
+    <br/><br/>
 
-## Mechanical Interface/Attachments to Rover:
-  * 1/2 inch clamping hub to 1/2 inch aluminum rod on the body
-  * 1/2 inch aluminum rod to 1/2 inch clamping hub on rocker-bogie arms
\ No newline at end of file
+    | ![15 inch rod cut](images/15inch_cut.PNG) | ![3 inch rod cut](images/3inch_cut.PNG)|
+    |:-:|:-:|
+    | Figure 1| Figure 2 |
+
+    <br/><br/>
+
+2. Aluminum Rods: Drilling
+    Parts/Tools Necessary
+    | Item | Ref | Qty | Image |
+    | :--- | :-- | :-- | :---: |
+    | 0.5" x 13.5" Aluminum Tube (Modified) | S16B | 1 | ![S16B](/images/components/Structural/S16B.png)|
+    | 0.5" x 3" Aluminum Tube (Modified) | S18A | 2 | ![S18A](/images/components/Structural/S18A.png)|
+    | Hand Drill or drill press | D3 | 1 |![D3](/images/components/Tools/D3.png)|
+    | Drill bit #23 | D6 | 1 | ![D6](/images/components/Tools/D6.jpeg)|
+    | Center punch or start drill bit | D7 | 1 |![D7](/images/components/Tools/D7.jpeg) |
+    | Vice or V-clamp | D8 | 1 | ![D8](/images/components/Tools/D8.png) | 
+
+
+    The turnbuckles must be attached to the differential pivot and rocker-bogie arm. We will accomplish this by drilling holes in the aluminum beam S16B and attaching two 5-hole aluminum bars on each side.
+    
+    1. Using a vice or clamp, firmly grab onto the 0.5 x 13.5 inch rod S16B with the end extending out around 2 inches from the edge of the vice/clamp. Mark the dimensions as shown in Figure 3. Carefully use a center drill to start the a center hole for these holes. It is important that the center hole is as centered as possible to prevent the bit from walk- ing/slipping during drilling, which could result in the bit breaking. Then, use a #23 (0.154 inch diameter) drill bit and drill all the way through both sides of the rod. This makes the S16B’ part.
```

Comment:
```suggestion
    1. Using a vice or clamp, firmly grab onto the 0.5 x 13.5 inch rod S16B with the end extending out around 2 inches from the edge of the vice/clamp. Mark the dimensions as shown in Figure 3. Carefully use a center drill to start the a center hole for these holes. It is important that the center hole is as centered as possible to prevent the bit from walking/slipping during drilling, which could result in the bit breaking. Then, use a #23 (0.154 inch diameter) drill bit and drill all the way through both sides of the rod. This makes the S16B’ part.
```